### ISSUE TITLE
[INTAKE] Update footer label and add new content to the accessibility pages (English and Spanish)

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#132323]
# Content and IA Product Support Request

## Team info

- **Product name:** Accessibility
- **Team name:** ADE 
- **Slack channel:** #accessibility-help
- **OCTO product owner:** @artsymartha68 @DanielleThierryUSDSVA 

 
### What audiences is your work for
- [x] Veterans and/or their family members or caregivers
- [ ] Other VA.gov audience
  - Please describe the audience:
- [ ] Internal designers and product team members (i.e. pattern or standards work)

### Which of these descriptions best fits the work we’ll partner on?

- [ ] Digitizing a new form
- [ ] Creating a new digital tool for Veterans or their beneficiaries
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [x] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Components, patterns, or general standards
- [ ] Something else:

### Describe the work effort

Per request of the VA business partners in the 508 Office and Office of Resolution Management, we need to make the following changes/additions:

1. Revert back to "Accessibility" as the link name in the standard footer(s) (currently it's "Section 508")
2. Change the H1 of the [accessibility page](https://www.va.gov/accessibility-at-va/) and the [Spanish version](https://www.va.gov/accessibility-at-va-esp/) to match the footer name 
3. Add new content to the [accessibility page](https://www.va.gov/accessibility-at-va/) and the [Spanish version](https://www.va.gov/accessibility-at-va-esp/) as well

### Supporting artifacts

This is the content we need to add to the accessibility pages (I don't have a Spanish translation!):

Architectural Barriers Act of 1968 (42 U.S.C. §§ 4151--57)

The Architectural Barriers Act (ABA) requires access to facilities that are designed, built, altered, or leased with Federal funds. The Access Board is the federal agency responsible for enforcing the ABA. The Access Board's accessibility standards are available on their website at [www.access-board.gov/guidelines-and-standards/buildings-and-sites/about-the-aba-standards](http://www.access-board.gov/guidelines-and-standards/buildings-and-sites/about-the-aba-standards).

If you believe that a physical facility that is designed, built, altered, or leased with Federal funds by U.S. Department of Veterans Affairs does not comply with the Architectural Barriers Act (ABA), refer to the US Access Board’s website under https://www.access-board.gov/enforcement/.
  
### Timeline

- **Estimated launch timeline:** As soon as you can, per our business partner there is an upcoming EEOC review (I don't have the date)
- **Where are you at in your design/development process?** n/a
- **Is a staged launch planned? If so, please provide details.** n/a

**Comment by tamika-matthews** [comment]
Including @rorycooper for awareness.

**Comment by jonichan-ia** [comment]
I've completed the IA tasks: changed "508 compliance" to "Accessibility" (that's what the revision history said a year ago). The changes are showing live now.

**Comment by artsymartha68** [comment]
I had a question about the two new external links which open in a new tab. Should they:
1. Match the existing external links on the page and open in the same tab
2. Open in a new tab, and if yes then add "(opens in a new tab)" to the link labels

I leave it up to y'all!

**Comment by laurenvocke-va** [comment]
CMS team said this may be a difference in the preview server and the production server. I can publish and we can test what the behavior is after it's live. If we need to make any changes after that I can go in quickly and do so.
